titanic.ipynb

```diff
@@ -8,6 +8,7 @@
         "## Titanic 3\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 Line 7_
```python
import seaborn as sns
import matplotlib.pyplot as plt

# Load the Titanic dataset
titanic = sns.load_dataset('titanic')

# Create a countplot of the man/woman/child classification
```
singel comment

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/10)._